### DISCUSSION TITLE
Are there any information or documentation details on how CLP works？

### DISCUSSION THREAD

**Discussion by tusiqi1** [coin-or/Clp#270] [Q&A] [answered]
Are there any information or documentation details on how CLP works？

**Comment by Iain-R** [comment, marked as answer]
Hi, if you are looking for documentation on the classes and how CLP can be used then there is documentation available in a Doxygen format, which can be found here : https://coin-or.github.io/Clp/Doxygen/
I also found the [examples](https://github.com/coin-or/Clp/tree/master/examples) helpful, the [user guide](https://www.coin-or.org/Clp/userguide/clpuserguide.html) explains the focus of each example.

**Reply by tusiqi1** [reply]
thank you!

**Reply by Iain-R** [reply]
If I can find any other useful resources or documentation I will let you know and post it here for others to find.
